src/components/Widgets/SubscriptionsWidget.tsx

```diff
@@ -3,20 +3,19 @@ import OutlinedCalendarAltIcon from '@patternfly/react-icons/dist/js/icons/outli
 import useStatus from '../../hooks/useStatus';
 import { QueryClient, QueryClientProvider } from 'react-query';
 import './SubscriptionsWidget.scss';
-import {
-  EmptyState,
-  EmptyStateVariant,
-  EmptyStateIcon,
-  EmptyStateBody,
-  Stack,
-  StackItem,
-  Title,
-  Alert,
-  Gallery
-} from '@patternfly/react-core';
+import { EmptyState } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateVariant } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateIcon } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateBody } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { Stack } from '@patternfly/react-core/dist/dynamic/layouts/Stack';
+import { StackItem } from '@patternfly/react-core/dist/dynamic/layouts/Stack';
+import { Title } from '@patternfly/react-core/dist/dynamic/components/Title';
+import { Alert } from '@patternfly/react-core/dist/dynamic/components/Alert';
+import { AlertVariant } from '@patternfly/react-core/dist/dynamic/components/Alert';
+import { Gallery } from '@patternfly/react-core/dist/dynamic/layouts/Gallery';
```

Comment:
Question: what's the motivation for moving from the bulk import to these?

Reply:
@jlgre I didn't do this. It happen automatically when I pushed the commit. I've not seen this before.

Reply:
Interesting - does it happen on push or on commit? I've never had that happen to my pull requests

Reply:
On push. I then immediately see that I need to rebase.

Reply:
I just committed changes restoring the imports, and the only thing that came through was a line break.

Reply:
I could try putting it back manually on GH.

Reply:
Yeah, I would give that a try.

It probably isn't that important, but I would rather it not change

Reply:
https://github.com/RedHatInsights/subscription-inventory-ui/pull/186 - it looks like I'm able to make a change in that file and the imports not reorder.

Wonder if its cause its a fork? It shouldn't though

Reply:
@jlgre Updated. I'll have another PR soon to update the Empty State styling. Thanks for your help.

Reply:
Sweet - what ended up to be causing this? Just curious cause I'd never seen it before

Reply:
@jlgre No clue - 10 years on GH and I've never had this happen.